drivers/SmartThings/matter-switch/src/third-reality/init.lua

```diff
@@ -0,0 +1,69 @@
+-- Copyright 2023 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+local log = require "log"
+local clusters = require "st.matter.clusters"
+local MatterDriver = require "st.matter.driver"
+local utils = require "st.utils"
+
+local THIRD_REALITY_MANUFACTURER_ID = 0x1407
+
+
+local function is_third_reality_products(opts, driver, device)
+  if device.manufacturer_info.vendor_id == THIRD_REALITY_MANUFACTURER_ID then
+    return true
+  end
+
+  return false
+end
+
+local function illuminance_attr_handler(driver, device, ib, response)
+  local lux = math.floor(10 ^ ((ib.data.value - 1) / 10000))
+  device:emit_event_for_endpoint(ib.endpoint_id, capabilities.illuminanceMeasurement.illuminance(lux))
+end
+
+local function occupancy_attr_handler(driver, device, ib, response)
+  device:emit_event(ib.data.value == 0x01 and capabilities.motionSensor.motion.active() or capabilities.motionSensor.motion.inactive())
+end
+
+local third_reality_handler = {
+  NAME = "Third Reality Handler",
+  matter_handlers = {
+    attr = {
+      [clusters.IlluminanceMeasurement.ID] = {
+        [clusters.IlluminanceMeasurement.attributes.MeasuredValue.ID] = illuminance_attr_handler
+      },
+      [clusters.OccupancySensing.ID] = {
+        [clusters.OccupancySensing.attributes.Occupancy.ID] = occupancy_attr_handler,
+      }
+    },
+  },
+  subscribed_attributes = {
```

Comment:
You will want to add all relevant subscribed attributes here, including switch, switchLevel, etc. as you had before. This way, the driver will continue to subscribe to those attributes and you will see all of these attributes update the capabilities in the app.
```
subscribed_attributes = {
    [capabilities.switch.ID] = {
      clusters.OnOff.attributes.OnOff
    },
    [capabilities.switchLevel.ID] = {
      clusters.LevelControl.attributes.CurrentLevel
    },
    [capabilities.colorControl.ID] = {
      clusters.ColorControl.attributes.CurrentHue,
      clusters.ColorControl.attributes.CurrentSaturation,
      clusters.ColorControl.attributes.CurrentX,
      clusters.ColorControl.attributes.CurrentY,
    },
    [capabilities.colorTemperature.ID] = {
      clusters.ColorControl.attributes.ColorTemperatureMireds,
    },
    [capabilities.illuminanceMeasurement.ID] = {
      clusters.IlluminanceMeasurement.attributes.MeasuredValue
    },
    [capabilities.motionSensor.ID] = {
      clusters.OccupancySensing.attributes.Occupancy
    }
  },
```